packages/devtools_app/lib/src/debugger/file_search.dart

```diff
@@ -168,32 +201,48 @@ class FileQuery {
   bool isFuzzyMatch(ScriptRef script) {
     if (isEmpty) return false;
 
-    final fileName = _fileName(script.uri);
-    return fileName.caseInsensitiveFuzzyMatch(query);
+    if (isMultiToken) {
+      return script.uri.caseInsensitiveFuzzyMatch(query.replaceAll(' ', ''));
+    }
+
+    return _fileName(script.uri).caseInsensitiveFuzzyMatch(query);
   }
 
   AutoCompleteMatch createFuzzyMatchAutoCompleteMatch(ScriptRef script) {
     if (isEmpty) return AutoCompleteMatch(script.uri);
 
-    final fileName = _fileName(script.uri);
-    final fileNameIdx = script.uri.lastIndexOf(fileName);
-    final matchedSegments = _findFuzzySegments(fileName)
-        .map((range) =>
-            Range(range.begin + fileNameIdx, range.end + fileNameIdx))
-        .toList();
+    List<Range> matchedSegments;
+    if (isMultiToken) {
+      matchedSegments =
+          _findFuzzySegments(script.uri, query.replaceAll(' ', ''));
+    } else {
+      final fileName = _fileName(script.uri);
+      final fileNameIdx = script.uri.lastIndexOf(fileName);
+      matchedSegments = _findFuzzySegments(fileName, query)
+          .map((range) =>
+              Range(range.begin + fileNameIdx, range.end + fileNameIdx))
+          .toList();
+    }
 
     return AutoCompleteMatch(script.uri, matchedSegments: matchedSegments);
   }
 
   List<Range> _findExactSegments(String file) {
-    final autoCompleteResultSegments = <Range>[];
-    final start = file.indexOf(query);
-    final end = start + query.length;
-    autoCompleteResultSegments.add(Range(start, end));
-    return autoCompleteResultSegments;
+    final matchedSegments = <Range>[];
+    for (final token in isMultiToken ? tokens : [query]) {
+      final autoCompleteResultSegments = <Range>[];
+      final start = file.indexOf(token);
+      final end = start + token.length;
+      autoCompleteResultSegments.add(Range(start, end));
+      matchedSegments.addAll(autoCompleteResultSegments);
```

Comment:
instead of creating the `autoCompleteResultSegments` variable, you can just add the Range to `matchedSegments` directly:

`matchedSegments.addd(Range(start, end));`